src/main/java/app/bpartners/geojobs/endpoint/rest/security/authentication/apikey/authorizer/CommunityZoneDetectionJobProcessAuthorizer.java

```diff
@@ -0,0 +1,54 @@
+package app.bpartners.geojobs.endpoint.rest.security.authentication.apikey.authorizer;
+
+import app.bpartners.geojobs.endpoint.rest.controller.mapper.DetectableObjectConfigurationMapper;
+import app.bpartners.geojobs.endpoint.rest.model.DetectableObjectConfiguration;
+import app.bpartners.geojobs.endpoint.rest.security.authentication.apikey.ApiKeyAuthenticationFilter;
+import app.bpartners.geojobs.model.exception.ForbiddenException;
+import app.bpartners.geojobs.repository.CommunityAuthorizationDetailsRepository;
+import app.bpartners.geojobs.repository.model.detection.DetectableType;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Predicate;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+public class CommunityZoneDetectionJobProcessAuthorizer
```

Comment:
Set it inside a global AuthProvider with the function that checks the apiKey. 
And configure SecurityConf globally to use the AuthProvider. So then, you don't need to specify explicitly in each controller.